tests/test_http_writer.py

```diff
@@ -262,6 +262,18 @@ async def test_write_to_closing_transport(
         await msg.write(b"After closing")
 
 
+async def test_write_to_closed_transport(
+    protocol: Any, transport: Any, loop: Any
+) -> None:
+    msg = http.StreamWriter(protocol, loop)
+
+    await msg.write(b"Before transport close")
+    protocol.transport = None
+
+    with pytest.raises(ConnectionResetError):
```

Comment:
Remember to always use `match=` with `pytest.raises()`.

Reply:
21bb53bbaed029df234a3961a7489d1b2977e4a6